### ISSUE TITLE
Scaffold plugin can not be tested

### ISSUE THREAD

**Issue by sourcerer-mike** [wp-cli/wp-cli#1921]
running phpunit leads to this error:

```
PHP Warning:  require_once(/tmp/wordpress-tests-lib/includes/functions.php): failed to open stream: No such file or directory 
```

In the plugin there is a "tests/bootstrap.php" which tries to open a non-existent file:

```
require_once $_tests_dir . '/includes/functions.php';
```

It could be that you try to create such file via "install-wp-tests.sh:59"

```
svn co --quiet http://develop.svn.wordpress.org/trunk/tests/phpunit/includes/ $WP_TESTS_DIR
```

But this has no "includes" directory.

1. Fixing "bootstrap.php" to the right path ends in more errors.
2. Fixing the install-wp-tests.sh brings up even more errors.

(`svn co --quiet http://develop.svn.wordpress.org/trunk/tests/phpunit/includes/ $WP_TESTS_DIR/includes`)

It would be nice to have a better / working testing environment.

**Comment by danielbachhuber** [comment]
Can you share `wp --info` ?

**Comment by sourcerer-mike** [comment]
Almost latest:

```
PHP binary:     /usr/bin/php
PHP version:    5.5.20
php.ini used:   /etc/php.ini
WP-CLI root dir:        .../Sites/foo/vendor/wp-cli/wp-cli
WP-CLI global config:   
WP-CLI project config:  
WP-CLI version: 0.19.0
```

**Comment by danielbachhuber** [comment]
This is fixed in v0.19.2. You should upgrade to the latest.
